Api.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace api;
```

Comment:
I also think `tooling` is a better namespace then (or `scripts` if the main purpose of your package is _to be_ a tool 😉)